src/components/MyHousesList/MyHousesList.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
It would be best if this was placed inside `components/ProfilePage` as all other components used exclusively by the my-profile page. You could just move the whole folder `MyHousesList` there.